chi/server.py

```diff
@@ -538,7 +540,7 @@ def wait_for_tcp(host, port, timeout=(60 * 20)):
             with socket.create_connection((host, port), timeout=timeout):
                 break
         except OSError as ex:
-            time.sleep(0.01)
+            time.sleep(sleep_time)
```

Comment:
Ah jeez, I did not realize it was such an incredibly short timeout 🤦🏻 . Man, that's awful. Definitely good to increase this significantly.